Doc/library/bdb.rst

```diff
@@ -311,16 +361,18 @@ The :mod:`bdb` module also defines two classes:
 
    .. method:: get_stack(f, t)
 
-      Get a list of records for a frame and all higher (calling) and lower
-      frames, and the size of the higher part.
+      Return a list of (frame, lineno) tuples in a stack trace and a size.
```

Comment:
Does this need a comma before _and a size_? Are the tuples in the size or just in the stack trace?